test/docker_tests/test_mount_diagnostics_pytest.py

```
@@ -0,0 +1,411 @@
+                    
+                except AssertionError as e:
+                    pytest.fail(f"Table validation failed for {test_scenario.name}: {e}")
+            
+            return  # Test passed - container correctly detected issues and exited
+
+            # Container is still running - run diagnostic tool
+            cmd_exec = [
+                "docker", "exec", "--user", "netalertx", container_name,
+                "python3", "/entrypoint.d/10-mounts.py"
+            ]
+            result_exec = subprocess.run(cmd_exec, capture_output=True, text=True, timeout=30)
```

Ruff (0.14.2)

`subprocess` call: check for execution of untrusted input

(S603)